src/storage/src/hummock/state_store.rs

```diff
@@ -335,7 +328,7 @@ impl HummockStorage {
         // See comments in HummockStorage::iter_inner for details about using compaction_group_id in
         // read/write path.
         assert!(pinned_version.is_valid());
-        for level in pinned_version.levels(compaction_group_id) {
+        for level in pinned_version.levels(table_id.table_id()) {
```

Comment:
How about acquire the `compaction_group_id` by `pinned_version` directly before we call `levels_by_compaction_group_id`. So that we no need to maintain a new function and pass table_id to it ?

Reply:
> How about acquire the `compaction_group_id` by `pinned_version` directly before we call `levels_by_compaction_group_id`. So that we no need to maintain a new function and pass table_id to it ?

I wrote like this before, but I found that in many tests we use table_id==0 to represent full `HummockVersion` scan. If we want to retain it, it's awkward to replace this with `get_compaction_group_id` plus `levels_by_compaction_group_id` logic

Reply:
Makes sense, let's consider what other reviewers have to say